### ISSUE TITLE
Workflow deletion stops working

### ISSUE THREAD

**Issue by norman-kapschefsky** [dmvict/clean-workflow-runs#3]
Hi,
since a week, somehow the action stops working - it just runs through but does not delete the configured workflow states anymore. Before that, everything worked fine and my prior skipped and cancelled workflows for a certain branch were deleted as expected. With "dry: true" I also see no workflows to delete, even though there are many ones listed in github actions web ui (for that branch). There is no error output from your action step.

Looks like the obtained workflow list is always zero. I dont know if something on github has changed internally that results in that behavior for instance.

I appreciate any help. Thank you in advance.

This is my configured step:
```
      - name: Delete skipped and canceled workflows for this branch
        uses: dmvict/clean-workflow-runs@v1.0.0
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          run_conclusions: |
            cancelled
            skipped
          save_min_runs_number: 0
          save_period: 0
          branch: ${{ steps.branch-name.outputs.current_branch }}
```

CI output
```
Run dmvict/clean-workflow-runs@v1.0.0
with:
token: ***
run_conclusions: cancelled
skipped

save_min_runs_number: 0
save_period: 0
branch: hf_123_feature-x-setup
dry: false
```

**Comment by dmvict** [comment]
Hello, @norman-kapschefsky. 

Thank you for your issue. 

I'll test the action and fix the problem but little later. The action is not my main work, it's like pet project with continuous support.

**Comment by norman-kapschefsky** [comment]
Hi @dmvict,
thank you for checking. I'm looking forward to that, but take your time 👍

**Comment by dmvict** [comment]
@norman-kapschefsky 

I've tested the action locally and have no failures. 

Then I run the next action code on the real repository
```yaml
      - name: Delete skipped and cancelled runs
        uses: dmvict/clean-workflow-runs@v1.0.0
        with:
          token: ${{ secrets.PRIVATE_GITHUB_TOKEN }}
          run_conclusions: |
            cancelled
            skipped
          save_period: 0
          save_min_runs_number: 0
          branch : doc
```

The workflow successfully deleted skipped and canceled workflows on branch `doc`.  [Link on test run](https://github.com/dmvict/wTools/runs/6004382639?check_suite_focus=true)

I suggest that your token have no permissions for deleting. 

Please, try another token and give me report.

**Comment by norman-kapschefsky** [comment]
Hi, thank you for investing time into that.

I tried a lot of things today and had strange results. But at the end it seems that it has nothing to do with your action. I just found a related bug report. It seems that the api for obtaining the workflows is not working like it should be at the moment. Several people reported the last hours/days that with added filter options the api does not respond as expected. It basically returns 0 workflows (especially for new branches o.O - maybe their index is out of control).

https://github.community/t/bug-github-api-list-workflow-runs-returns-0-runs-even-though-there-are-runs/165080/12

I guess thats the real issue where we have to wait for an api fix. But anyhow, your code shouldn't have any problems so feel free to close my issue.

Thanks again for your quick response and your effort!
